src/main/java/frc/team2412/robot/subsystems/LauncherSubsystem.java

```diff
@@ -76,63 +80,101 @@ public class LauncherSubsystem extends SubsystemBase {
+		launcherAngleOneMotor =
+				new CANSparkFlex(Hardware.LAUNCHER_PIVOT_ONE_MOTOR_ID, MotorType.kBrushless);
+		launcherAngleTwoMotor =
+				new CANSparkFlex(Hardware.LAUNCHER_PIVOT_TWO_MOTOR_ID, MotorType.kBrushless);
 		// encoders
 		launcherTopEncoder = launcherTopMotor.getEncoder();
 		launcherBottomEncoder = launcherBottomMotor.getEncoder();
-		launcherAngleEncoder = launcherAngleMotor.getAbsoluteEncoder(Type.kDutyCycle);
+		launcherAngleEncoder = launcherAngleOneMotor.getAbsoluteEncoder(Type.kDutyCycle);
 
 		// PID controllers
 		// Create launcherTopPIDController and launcherTopMotor]
 		launcherTopPIDController = launcherTopMotor.getPIDController();
 		launcherTopPIDController.setFeedbackDevice(launcherTopEncoder);
 		launcherBottomPIDController = launcherBottomMotor.getPIDController();
 		launcherBottomPIDController.setFeedbackDevice(launcherBottomEncoder);
-		launcherAnglePIDController = launcherAngleMotor.getPIDController();
-		launcherAnglePIDController.setFeedbackDevice(launcherAngleEncoder);
+		launcherAngleOnePIDController = launcherAngleOneMotor.getPIDController();
+		launcherAngleOnePIDController.setFeedbackDevice(launcherAngleEncoder);
+		// launcherAngleTwoPIDController = launcherAngleTwoMotor.getPIDController();
+		// launcherAngleTwoPIDController.setFeedbackDevice(launcherAngleEncoder);
+
 		Shuffleboard.getTab("Launcher")
-				.add(new SparkPIDWidget(launcherAnglePIDController, "launcherAnglePIDController"));
+				.add(new SparkPIDWidget(launcherAngleOnePIDController, "launcherAngleOnePIDController"));
+		// Shuffleboard.getTab("Launcher")
+		//		.add(new SparkPIDWidget(launcherAngleTwoPIDController, "launcherAngleTwoPIDController"));
 		Shuffleboard.getTab("Launcher")
 				.add(new SparkPIDWidget(launcherTopPIDController, "launcherTopPIDController"));
 		Shuffleboard.getTab("Launcher")
 				.add(new SparkPIDWidget(launcherBottomPIDController, "launcherBottomPIDController"));
+
+		configMotors();
 	}
 
 	public void configMotors() {
 		launcherTopMotor.restoreFactoryDefaults();
 		launcherBottomMotor.restoreFactoryDefaults();
-		launcherAngleMotor.restoreFactoryDefaults();
+		launcherAngleOneMotor.restoreFactoryDefaults();
+		launcherAngleTwoMotor.restoreFactoryDefaults();
 		// idle mode (wow)
 		launcherTopMotor.setIdleMode(IdleMode.kCoast);
 		launcherBottomMotor.setIdleMode(IdleMode.kCoast);
-		launcherAngleMotor.setIdleMode(IdleMode.kBrake);
+		launcherAngleOneMotor.setIdleMode(IdleMode.kBrake);
+		launcherAngleTwoMotor.setIdleMode(IdleMode.kBrake);
 		// inveritng the bottom motor lmao
 		launcherBottomMotor.setInverted(true);
 
 		// current limit
-		launcherTopMotor.setSmartCurrentLimit(20);
-		launcherBottomMotor.setSmartCurrentLimit(20);
-		launcherAngleMotor.setSmartCurrentLimit(20);
+		launcherTopMotor.setSmartCurrentLimit(40);
+		launcherBottomMotor.setSmartCurrentLimit(40);
+		launcherAngleOneMotor.setSmartCurrentLimit(20);
```

Comment:
We should disable one of the angle motors when first testing the pivot so the motors do not fight each other and burn one out.